Description: Don't install mailq
 Upstream only accepts the "mailq" shortcut to "smtpctl show" for historical
 reasons; the proper way is to use "smtpctl show".
 .
 Administrators who insist on having mailq can create a shell alias, or a
 symlink from mailq to smtpctl.
Author: Ryan Kavanagh <rak@debian.org>
Origin: vendor
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: opensmtpd/mk/smtpctl/Makefile.am
===================================================================
--- opensmtpd.orig/mk/smtpctl/Makefile.am	2013-12-23 09:58:00.056592026 -0500
+++ opensmtpd/mk/smtpctl/Makefile.am	2013-12-23 09:58:19.469242900 -0500
@@ -66,7 +66,6 @@
 install-exec-hook: $(CONFIGFILES) $(MANPAGES)
 	$(MKDIR_P) $(DESTDIR)$(mandir)/$(mansubdir)8
 	if test "$(USE_MAILWRAPPER)" = "0"; then \
-		ln -s $(sbindir)/smtpctl  $(DESTDIR)$(sbindir)/mailq;\
 		ln -s $(sbindir)/smtpctl  $(DESTDIR)$(bindir)/sendmail;\
 	fi
 	$(INSTALL) -m 644 smtpctl.8.out		$(DESTDIR)$(mandir)/$(mansubdir)8/smtpctl.8
@@ -78,6 +77,5 @@
 	rm -f	$(DESTDIR)$(mandir)/$(mansubdir)8/sendmail.8			\
 	rmdir	$(DESTDIR)$(mandir)/$(mansubdir)8 2> /dev/null || /bin/true
 	if test "$(USE_MAILWRAPPER)" = "0"; then \
-		rm -f $(DESTDIR)$(sbindir)/mailq;\
 		rm -f $(DESTDIR)$(bindir)/sendmail;\
 	fi
